public/electron.js

```diff
@@ -82,15 +82,39 @@ function initialize(queues) {
   )
 
   mainWindow.webContents.on('did-finish-load', () => mainWindow.show())
-  // mainWindow.webContents.openDevTools()
+
+  if (process.env.PUBLISH50_DEV_TOOLS) {
+    mainWindow.webContents.openDevTools()
+  }
+
   mainWindow.on('closed', () => {
     mainWindow = null
   })
 }
 
 app.on('ready', async () => {
 
-  globalShortcut.register('CommandOrControl+Q', () => app.quit())
+  globalShortcut.register('CommandOrControl+Q', () => {
+
+    // Prompt user before closing the application if there are active jobs
+    if (Object.keys(queues['queues']).some((job) => {
```

Comment:
Let's rename `job` to `qname` because technically it doesn't store a job but a queue name.